packages/SwingSet/test/upgrade/bootstrap-scripted-upgrade.js

```diff
@@ -282,5 +282,26 @@ export const buildRootObject = () => {
 
       return [paramA, paramB];
     },
+
+    buildV1WithVatParameters: async () => {
+      const bcap1 = await E(vatAdmin).getNamedBundleCap('ulrik1');
+      const vp1 = { number: 1, marker };
+      const options1 = { vatParameters: vp1 };
+      const res = await E(vatAdmin).createVat(bcap1, options1);
+      ulrikAdmin = res.adminNode;
+      ulrikRoot = res.root;
+      const param1 = await E(ulrikRoot).getParameters();
+      return param1;
```

Comment:
No, quite the opposite, it's checking that the created/upgraded vat received the correct parameters. If this bootstrap vat returned `marker` then we'd be bypassing the created/upgraded vat entirely.

The test is actually preparing to check an additional feature, not implemented in this PR, which is to change the `upgrade()` semantics from "no `vatParameters` means `{}`" to "no `vatParameters` means reuse the retained value". Since the immediate goal is *preservation* of the old vP, the subsequent goal will be to enable *unilateral* upgrade (where vat-admin isn't even involved, much less userspace calling `upgrade()`), neither required a change to the `upgrade()` API. I realized I could/should reduce the scope of this PR after writing the tests but before writing the implementation. So the test is all set up to check that an `upgrade()` without `options.vatParameters` will return the old marker, and applies the correct assertion, but still supplies the option to the `upgrade()` call.

Reply:
Sorry I meant should this "also" return the marker, so that the test file can compare the identities.